src/ui/SelectionTable.js

```diff
@@ -61,69 +72,120 @@ const SelectionTable = ({ columns, data, onSelectionChanged, ...props }) => {
     ]);
   });
 
+  const { t } = useTranslation();
```

Comment:
I agree with @geoffreydhuyvetters 's comment on the original PR that we should not use the translations hook in the component library, but instead pass them down as props. Can either be strings in the case of simple translations, or functions in the case of translations like here with the dynamic count. So for example a `renderSelectedRowCountText` function prop that you can call like `renderSelectedRowCountText(count)` (not 100% sure if that's the best name though, just an example). Maybe @geoffreydhuyvetters also has some more input on this.